examples/convert_ieeg_to_bids.py

```diff
@@ -144,12 +108,13 @@
 # - `background on FreeSurfer`_
 # - `MNE-Python coordinate frames`_
 #
-# Currently, MNE-Python supports the ``mni_tal`` coordinate frame, which
-# corresponds to the ``fsaverage`` BIDS coordinate system. All other coordinate
-# frames in MNE-Python if written with :func:`mne_bids.write_raw_bids` are
-# written with coordinate system ``'Other'``. Note, then we suggest using
-# :func:`mne_bids.update_sidecar_json` to update the sidecar
-# ``*_coordsystem.json`` file to add additional information.
+# Currently, MNE-Python supports the ``mni_tal`` and ``mri`` coordinate frames,
+# corresponding to the ``fsaverage`` and ``ACPC`` BIDS coordinate systems
+# respectively. All other coordinate coordinate frames in MNE-Python if written
```

Comment:
I don't think that `MRI` in corresponds to `ACPC` BIDS coordinate frame in general. In general, `MRI` corresponds to FreeSurfer surface RAS / tkRAS. This will only be `ACPC` if the MRI has been processed so that FreeSurfer surface RAS is ACPC. So this dataset seems to be the exception to the rule, but this is phrased as if it's always the case...

Reply:
I think that's explained pretty clearly immediately above

```
# %%
# When the locations of the channels in this dataset were found
# in `Locating Intracranial Electrode Contacts
# <https://mne.tools/dev/auto_tutorials/clinical/10_ieeg_localize.html>`_,
# the T1 was aligned to ACPC. So, this montage is in an
# `ACPC-aligned coordinate system
# <https://surfer.nmr.mgh.harvard.edu/fswiki/CoordinateSystems>`_.
# We can either save the channel positions in the subject's anatomical
# space (from their T1 image) or we can transform to a template space
# such as ``fsaverage``. To save them in the individual space, it is
# required that the T1 have been aligned to ACPC and then the channel positions
# be in terms of that coordinate system. Automated alignment to ACPC has not
# been implemented in MNE yet, so if the channel positions are not in
# an ACPC-aligned coordinate system, using a template (like ``fsaverage``)
# is the best option.
```

Reply:
Agreed but reading *this section* the statement sounds global. I think it needs "for this dataset" or "for a ACPC-aligned dataset" or something

Reply:
Sure just changed